### ISSUE TITLE
fine-tune: huggingface tranformers.Trainer.train() hangs with Llama3 base model

### ISSUE THREAD

**Issue by yangyangyyy123** [meta-llama/llama3#137]
https://github.com/huggingface/transformers/issues/30399

System Info
transformers version : '4.39.2'

Who can help?
we used our existing fine-tune code, which worked with llama1 and llama2 base models

    trainer = Trainer(
        model=model,
        tokenizer=tokenizer,
        args=training_args,
        **data_module,
        callbacks=[ManifoldTensorBoardLoggerCallback()],
    )
    trainer.train()
but once the trainer starts fine-tuning from a llama3-8B, it barely makes any progress ("only prints the 0% on the progress status once, and then never updates it) after 5 hours. previously with llama2-7B, it runs through 40% of our examples within 25 minutes

I see that it's the same code as recommended by wiki
https://huggingface.co/docs/transformers/training

**Comment by HamidShojanazeri** [comment]
@yangyangyyy123 the support for llama3 is only added to the latest transformers version can you pls upgrade to `4.40.1`

**Comment by yangyangyyy123** [comment]
@HamidShojanazeri  yes our team realized that and I updated to 4.40.0 last night, still had same issue, is 4.40.1 necessary vs 4.40.0 ?
